dvc/repo/ls_url.py

```diff
@@ -0,0 +1,35 @@
+from dvc.exceptions import URLMissingError
+from dvc.fs import get_cloud_fs
+
+
+def parse_external_url(url):
+    fs_cls, fs_config, fs_path = get_cloud_fs(None, url=url)
+    fs = fs_cls(**fs_config)
+    return fs, fs.path.abspath(fs.path.normpath(fs_path))
```

Comment:
Hm, why do we need abspath+normpath?

Reply:
I don't know, that's what the existing code is doing. Normally, `abspath(normpath(...))` should be equivalent to `abspath(...)` but I don't trust it to hold in practice.

Reply:
Right, that's more of a leftover in the `output`. You shouldn't need to do abspath nor normpath here.